### ISSUE TITLE
Error in OpenMP.sublime-syntax and error in importing requests.

### ISSUE THREAD

**Issue by phn** [Nevensky/SublimeFortran#15]
Hello,

I get two errors on startup using the latest version (as of 2016/06/30), 
but the plugin does work.

1)

```
no such context expression at line 276 column 20 in Packages/Fortran/grammars/FortranModern.sublime-syntax
Error loading Packages/SublimeFortran/grammars/OpenMP.sublime-syntax: Unable to read Packages/SublimeFortran/grammars/OpenMP.sublime-syntax in Packages/Fortran/grammars/FortranModern.sublime-syntax
first paint time: 0.27135
```

Perhaps due to "Packages/SublimeFortran/" instead of 
"Packages/Fortran/".

2)

```
reloading plugin Fortran.InlineLatexHover
Traceback (most recent call last):
  File "/home/phn/bin/sublime_text_3/sublime_plugin.py", line 78, in reload_plugin
    m = importlib.import_module(modulename)
  File "./python3.3/importlib/__init__.py", line 90, in import_module
  File "<frozen importlib._bootstrap>", line 1584, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1565, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1532, in _find_and_load_unlocked
  File "/home/phn/bin/sublime_text_3/sublime_plugin.py", line 658, in load_module
    exec(compile(source, source_path, 'exec'), mod.__dict__)
  File "InlineLatexHover in /home/phn/.config/sublime-text-3/Installed Packages/Fortran.sublime-package", line 5, in <module>
ImportError: No module named 'requests'
reloading plugin Fortran.linter
```

Regards,
Prasanth

**Comment by 315234** [comment]
Are these errors in the console or an intrusive popup on startup?

**Comment by phn** [comment]
Hello,

All in the console. And as I mentioned, the plugin continues to work.

I was looking to figure out some other issue and noticed these messages.

Regards,
Prasanth

**Comment by 315234** [comment]
Thanks for the report. There were actually 3 errors here:
- Can't find OpenMP syntax file due to wrong path
- `expression` context was not included in commit
- `requests` is not a default Python package

All 3 should be fixed in v1.0.14. I knew I should have avoided a version number ending in 13.

**Comment by phn** [comment]
Hello,

Thanks for the quick fixes!

Regards,
Prasanth
